api/openapi/v1/core-command.yaml

```diff
@@ -221,9 +213,9 @@ paths:
       description: Test service providing an indication that the service is available.
       responses:
         200:
-          description: pong as a string
+          description: Pong as a string
```

Comment:
Let's remove implementation detail and change this to "String literal confirming the service is up."